oshmem/shmem/c/shmem_clear_lock.c

```diff
@@ -27,5 +28,10 @@
 
 void shmem_clear_lock(volatile long *lock)
 {
-    _shmem_clear_lock((void *)lock, sizeof(long));
+	if (oshmem_shmem_mcs_locks) {
```

Comment:
general comment: pls use 4 space tabs as per [OMPI Coding Style](https://docs.open-mpi.org/en/main/developers/source-code.html#code-style)

Reply:
I have checked and fixed all files updated in this PR to stick to 4 space tabs.

Reply:
can you please double check? E.g. looks like 8-width tabs are used in this file (and possibly others)